e2e/util/context.go

```diff
@@ -0,0 +1,123 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubectl/pkg/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	// Placement
+	ocmclusterv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
+	// ManagedClusterSetBinding
+	ocmclusterv1beta2 "open-cluster-management.io/api/cluster/v1beta2"
+
+	ramen "github.com/ramendr/ramen/api/v1alpha1"
+	subscription "open-cluster-management.io/multicloud-operators-subscription/pkg/apis"
+	placementrule "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/placementrule/v1"
+)
+
+var (
+	KubeconfigHub string
+	KubeconfigC1  string
+	KubeconfigC2  string
+)
+
+type Cluster struct {
+	K8sClientSet *kubernetes.Clientset
+	CtrlClient   client.Client
+}
+
+type Context struct {
```

Comment:
util.Context conflicts in name with context.Context, should we consider renaming it to something else?

Reply:
I think this should be fine. It will always be used as util.Ctx so will not cause confusion. And I could not think out a better name. 

util.Context contains a global logger, and k8s clients to each k8s.